app/assets/javascripts/Measure.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require('mongoose');
+const Code = require('./basetypes/Code');
+const Interval = require('./basetypes/Interval');
+const Quantity = require('./basetypes/Quantity');
+
+const [Integer, Array, String, Boolean, Mixed, ObjectId, Date] = [
+  mongoose.Schema.Types.Integer,
```

Comment:
I made a similar comment on @ljades https://github.com/projecttacoma/cqm-models/pull/11, but I'm not sure `Integer` is a thing with Mongoose. I think it should be `Number`.

Reply:
I just spoke with @ljades about this, I think you took some of this from something I did for the QDM models (which I made a mistake in). You should remove `Integer` and `Float` and just use `Number` for Mongoose models. The Mongoid ones should be left as is though.